src/main/frontend/pipeline-graph-view/pipeline-graph/main/support/labels.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+
+import { nodeStrokeWidth } from '../support/StatusIcons';
+import { TruncatingLabel } from '../support/TruncatingLabel';
+import {
+    NodeLabelInfo,
+    LayoutInfo,
+    StageInfo,
+} from '../PipelineGraphModel';
+import { sequentialStagesLabelOffset } from '../PipelineGraphLayout';
+
+interface RenderBigLabelProps {
+	details: NodeLabelInfo;
+	layout: LayoutInfo;
+	measuredHeight: number;
+	selectedStage?: StageInfo;
+	isStageSelected: (stage?: StageInfo) => boolean;
+}
+/**
+ * Generate the Component for a big label
+ */
+export function BigLabel({
+	details,
+	layout,
+	measuredHeight,
+	isStageSelected,
+	selectedStage
+}: RenderBigLabelProps) {
+	/**
+	 * Is this any child of this stage currently selected?
```

Comment:
unlikely we even need this function in future, as we have logs on a different page
```suggestion
	 * Is any child of this stage currently selected?
```